packages/semantic-tokens/src/control/tokens.ts

```diff
@@ -411,28 +425,28 @@ export const materialAcrylicBlur = `var(${materialAcrylicBlurRaw})`;
 export const materialMicaBlur = `var(${materialMicaBlurRaw})`;
 export const iconThemeCtrlDefaultRest = `var(${iconThemeCtrlDefaultRestRaw})`;
 export const iconThemeCtrlDefaultSelected = `var(${iconThemeCtrlDefaultSelectedRaw})`;
-export const statusBrandTintBackground = `var(${statusBrandTintBackgroundRaw})`;
-export const statusBrandTintStroke = `var(${statusBrandTintStrokeRaw})`;
+export const statusBrandTintBackground = `var(${statusBrandTintBackgroundRaw}, ${colorBrandBackground2})`;
+export const statusBrandTintStroke = `var(${statusBrandTintStrokeRaw}, ${colorBrandStroke2})`;
 export const statusDangerBackground = `var(${statusDangerBackgroundRaw}, ${colorPaletteRedBackground3})`;
-export const statusDangerTintBackground = `var(${statusDangerTintBackgroundRaw})`;
-export const statusDangerTintStroke = `var(${statusDangerTintStrokeRaw})`;
+export const statusDangerTintBackground = `var(${statusDangerTintBackgroundRaw}, ${colorPaletteRedBackground1})`;
+export const statusDangerTintStroke = `var(${statusDangerTintStrokeRaw}, ${colorPaletteRedBorder1})`;
 export const statusDangerTintForeground = `var(${statusDangerTintForegroundRaw}, ${colorPaletteRedForeground3})`;
-export const statusWarningBackground = `var(${statusWarningBackgroundRaw}, ${colorPaletteDarkOrangeBackground3})`;
-export const statusWarningTintBackground = `var(${statusWarningTintBackgroundRaw})`;
-export const statusWarningTintStroke = `var(${statusWarningTintStrokeRaw})`;
-export const statusWarningTintForeground = `var(${statusWarningTintForegroundRaw})`;
+export const statusWarningBackground = `var(${statusWarningBackgroundRaw}, ${colorStatusWarningBackground3})`;
```

Comment:
I think we want to keep the yellow 'colorPaletteYellowBackground3' as the fallback for badge warning (and the equivalent in foreground/tint etc.).

When using legacy tokens, there should be no visual regressions from the original implementation.
When using semantic-tokens, users are free to defined statusWarningBackground as 'orange' in their theme.

If we want to update the legacy designs to use orange instead, we can tackle that separately once semantic migration is complete, for now if users want to update to orange, the new semantic definitions provide them the avenue to do either orange or yellow.